backend/api/urls.py

```diff
@@ -1,10 +1,15 @@
 from django.urls import path
 from . import views
+from rest_framework_simplejwt.views import (
+    TokenObtainPairView,
+    TokenRefreshView,
+)
 
 urlpatterns = [
-    path('', views.getRoutes),
-    path('users/register/', views.registerUser),
-    path('users/authenticate/', views.authenticateUser),
-    path('users/<int:pk>/delete/', views.deleteUser),
-    path('users/<int:pk>/', views.getUser),
+    path('', views.GetRoutes.as_view()),
+    path('users/register/', views.RegisterUser.as_view()),
+    path('users/delete/', views.DeleteUser.as_view()),
+    path('users/get/', views.GetUser.as_view()),
+    path('users/login', TokenObtainPairView.as_view(), name='token_obtain_pair'),
+    path('users/authenticate/', TokenRefreshView.as_view(), name='token_refresh'),
```

Comment:
```suggestion
    path('users/login/', TokenObtainPairView.as_view(), name='token_obtain_pair'),
    path('users/authenticate/', TokenRefreshView.as_view(), name='token_refresh'),
```

add the trailing / for a unified http request url

Reply:
done